code/modules/clothing/under/miscellaneous.dm

```diff
@@ -291,56 +233,6 @@
 	worn_state = "dress_fire"
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO
 
-/obj/item/clothing/under/dress/dress_fire
```

Comment:
theres nothing wrong with dresses, not everyone wears uniform, especially if off duty. These might be used for some roleplaying fun